docs/request-handling-process.md

```diff
@@ -0,0 +1,34 @@
+# Handling requests
+
+The process for handling requests is somewhat complex and built to require
+maintaining as little state as possible, preferring to store messages in an
+SQS Queue over a long-term storage mechanism like a DynamoDB table.
+
+## Data Transformations
+
+The ATAT CSP API and the HOTH API do not use the same representations of data.
+This requires transformations between the data types during the requests.
+
+## Cost Request process
+
+This diagram covers the process for how the API is invoked for a Cost Request.
+
+```mermaid
```

Comment:
Well that's pretty neat.  [Mermaid](https://mermaid-js.github.io/)

Reply:
https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/ may be helpful too!